src/github.ts

```diff
@@ -336,6 +337,8 @@ export class GitHub {
         },
         throttle: {
           enabled: throttlingRetries > 0,
+          retryAfterBaseValue: 3000, // 3 seconds
```

Comment:
This is the number that will be multiplied with headers `retry-after` or `x-ratelimit-reset`. So we are multiplying by 3 what the server tells us.